tests/js/spec/views/groupEvents.spec.jsx

```diff
@@ -62,20 +62,35 @@ describe('groupEvents', function() {
     });
   });
 
-  it('can change environment', function() {
-    const component = shallow(
-      <GroupEvents
-        params={{orgId: 'orgId', projectId: 'projectId', groupId: '1'}}
-        location={{query: {}}}
-        environment={TestStubs.Environments()[0]}
-      />,
-      {
-        context: {...TestStubs.router(), group: TestStubs.Group()},
-        childContextTypes: {
-          router: PropTypes.object,
-        },
-      }
-    );
-    component.setProps({environment: TestStubs.Environments()[1]});
+  fdescribe('changing environment', function() {
```

Comment:
```suggestion
  describe('changing environment', function() {
```

We should use `.only` since the linters can catch that

Reply:
Would these tests fail on master?